butler-bot/bot.js

```diff
@@ -27,11 +27,9 @@ client.once('ready', () => {
 })
 
 client.on('guildMemberAdd', (member) => {
-  member.addRole(DESCRIBE_YOURSELF_ROLE_ID)
-    .then(() => { logger.info('Successfully added the role') })
-
-  if (member.user.bot) {
-    return
+  if (!member.user.bot) {
```

Comment:
Just return in this case. Otherwise, you won't assign any role to a bot, however you will send a message in the greetings channel.